bin/run.js

```diff
@@ -27,6 +27,12 @@ try {
 }
 
 // process runfile.js
-console.log('Processing runfile...')
-var runfile = require(path.resolve('./runfile'));
+try {
+    console.log('Processing runfile...')
+    var runfile = require(path.resolve('./runfile'));
+} catch(e) {
+  console.log('No runfile.js is defined.');
```

Comment:
It would be good to include also current working directory within the message, for example: `console.log('No runfile.js is defined within ${process.cwd()}'`). What do you think ?